### ISSUE TITLE
Suggestion : Afficher un ID pour chaque tâche

### ISSUE THREAD

**Issue by Fabrice61** [HUB-HUmanBeing/SystemD#213]
Ce serait hyper pratique d'avoir un ID (de type #) pour les tâches ne serait-ce que pour discuter autour d'une tâche. C'est plus facile de dire : "A propos de la tâche # 15, blabla bla..." que de donner le nom de la tâche...

Bon je sais que du coup cela ressemble à de nombreux outils de gestion de tâches/issues (comme Github) mais bon c'est quand même hyper pratique...

**Comment by robinBanquo** [comment]
ouais, pourquoi pas, pour l'instant l'id c'est celui en bdd, mais on pourrait faire en sorte d'avoir un compteur comme sur les issues github... l'idéal serait de pouvoir apres faire le lien automatique vers la tache a chaque fois qu'on a un #{chiffre} dans un contenu textuel du projet.

**Comment by Fabrice61** [comment]
Oui, tout à fait, c'est ce à quoi je pensais...et si la tâche est fermée la référence à l'ID est barrée du style : ![Capture du 2020-01-16 11-52-52](https://user-images.githubusercontent.com/59598652/72518977-c31caa00-3856-11ea-841e-904cb6197660.png)
